browser/main/lib/RcParser.js

```diff
@@ -0,0 +1,23 @@
+import _ from 'lodash'
+
+const path = require('path')
+const sander = require('sander')
+
+function parse (boostnotercPath) {
+  if (!sander.existsSync(boostnotercPath)) return
+  let config = JSON.parse(sander.readFileSync(boostnotercPath).toString())
+  return config
+}
+
+function exec (boostnotercPath) {
+  const config = this.parse(boostnotercPath)
+  if (config.execs === undefined) return
+  _.forEach(config.execs, (exec) => {
+    eval(exec)
```

Comment:
It's not `plugin`. Just I'd like to run some codes on starting Boostnote by `execs`.